airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/jdbc/DatabricksStorageOperations.kt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.databricks.jdbc
+
+import com.databricks.sdk.WorkspaceClient
+import io.airbyte.cdk.integrations.destination.record_buffer.SerializableBuffer
+import io.airbyte.integrations.base.destination.typing_deduping.DestinationHandler
+import io.airbyte.integrations.base.destination.typing_deduping.Sql
+import io.airbyte.integrations.base.destination.typing_deduping.SqlGenerator
+import io.airbyte.integrations.base.destination.typing_deduping.StreamConfig
+import io.airbyte.integrations.base.destination.typing_deduping.StreamId
+import io.airbyte.integrations.base.destination.typing_deduping.TypeAndDedupeTransaction
+import io.airbyte.integrations.base.destination.typing_deduping.migrators.MinimumDestinationState
+import io.airbyte.integrations.destination.sync.StorageOperations
+import io.airbyte.protocol.models.v0.DestinationSyncMode
+import io.github.oshai.kotlinlogging.KotlinLogging
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.util.*
+
+class DatabricksStorageOperations(
+    private val sqlGenerator: SqlGenerator,
+    private val destinationHandler: DestinationHandler<MinimumDestinationState.Impl>,
+    private val workspaceClient: WorkspaceClient,
+    private val database: String,
+    private val purgeStagedFiles: Boolean = false
+) : StorageOperations {
+
+    private val log = KotlinLogging.logger {}
+
+    // TODO: There are 2 methods used from SqlGenerator which were spread across in old code.
+    //  Hoist them to SqlGenerator interface in CDK, until then using concrete instance.
+    private val databricksSqlGenerator = sqlGenerator as DatabricksSqlGenerator
```

Comment:
which 2 methods is this needed for? (I think it's create+truncateRawTable?)

Reply:
Yeah those 2 i think, to decouple dependency from old SqlOps intf